tests/test_maintel_raise_m1m3.py

```diff
@@ -0,0 +1,60 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+import logging
+import random
+import unittest
+
+from lsst.ts.observatory.control.mock import MTCSMock
+
+from lsst.ts import standardscripts
+from lsst.ts.standardscripts.maintel.m1m3 import RaiseM1M3
+
+random.seed(47)  # for set_random_lsst_dds_partition_prefix
+
+logging.basicConfig()
+
+
+class TestRaiseM1M3(
+    standardscripts.BaseScriptTestCase, unittest.IsolatedAsyncioTestCase
+):
+    async def basic_make_script(self, index):
+        self.script = RaiseM1M3(index=index)
+        self.mtcs_mock = MTCSMock()
```

Comment:
I think you can remove the `MTCSMock`.. instead, you could do something like this:

```
self.script.mtcs.raise_m1m3 = unittest.mock.AsyncMock()
```